packages/config-generator/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "config-generator",
+  "version": "1.0.0",
```

Comment:
since this is private package and it not meant to be published, it is better to remove unnecessary fields and set `"private": true`